### DISCUSSION TITLE
Cas de Mayotte : donner l'information qu'il n'est pas possible d'y faire de diagnostic

### DISCUSSION THREAD

**Discussion by smdsgn** [MTES-MCT/sparte#368] [Idées]
Ajouter un encart d'information pour indiquer qu'il n'est pas possible de faire de diagnostic pour Mayotte.

**Comment by alexisig** [comment]
Mettre à jour les résultats de l'autocomplete pour ajouter Mayotte (et autres DROM-COM, sauf guyanne, martinique et guadeloupe) et indiquer "Diagnostic non disponible" (ou autre mention plus pertinente), et ajouter un lien vers la FAQ via un bouton intitulé "Pourquoi ?"
